src/SpeedManager.cc

```diff
@@ -33,7 +33,7 @@ SpeedManager::~SpeedManager()
 void SpeedManager::updateSpeed()
 {
 	speed = (fastforwardSetting.getBoolean() ? fastforwardSpeedSetting : speedSetting)
-	        .getInt() * (1.0 / 100.0);
+	        .getFloat() / 100.0;
```

Comment:
Please revert this. There's a technical reason why we wrote it like this. See https://godbolt.org/z/6hfee6GPn.

We want a multiplication here. But the compiler is not allowed to transform this division into multiplication by itself, because this may have different rounding behavior (because the extra 1.0 / 100.0 also gets rounded).

In the past we compiled with `-ffast-math`. That does give permission to the compiler for these transformation. But it gave problems in some Dear ImGui calculation.

Reply:
I don't know why I'm seeing it here, but I did so in my last force-push....

Reply:
OK, apparently I only updated the commitmessage and I did not commit the actual changes... fixed now. Sorry about the fuss.